hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/contracts/ContractsModule.java

```diff
@@ -211,4 +214,15 @@ static Supplier<CallLocalEvmTxProcessor> provideCallLocalEvmTxProcessorFactory(
         return () -> new CallLocalEvmTxProcessor(
                 codeCache, livePricesSource, dynamicProperties, gasCalculator, mcps, ccps, aliasManager);
     }
+
+    @Provides
+    @Singleton
+    @Named("PrecompileDetector")
+    static Predicate<Address> providePrecompileDetector() {
+        // Addresses mapping to 0.0.0 < x < 0.0.751 are never calling the Hedera account.
+        return address -> {
+            final var numberOfLeadingZeroBytes = address.numberOfLeadingZeroBytes();
+            return numberOfLeadingZeroBytes >= 18 && numberOfLeadingZeroBytes < Address.SIZE && Integer.compareUnsigned(address.getInt(16), 750) <= 0;
```

Comment:
0.0.0 should be included in the list.  That streamlines the labda

Reply:
Added!